plugin/al/translators/tests/testTranslators.py

```diff
@@ -132,6 +134,52 @@ def assertUsingUsdVariant():
         # ...and then make sure that our ref edit was preserved
         self.assertEqual(mc.getAttr('cubeNS:pCube1.translate')[0], (4.0, 5.0, 6.0))
 
+    def testMayaReference_usdPrimNamespace(self):
+        """
+        Tests that connection between MayaReference prim and maya reference does not
+        break on hierarchy changes if mayaNamespace UsdAttribute is used.
+        """
+        import AL.usdmaya
+
+        mc.file(new=1, f=1)
+
+        # copy usd file A to temporary location
+        tempPath = tempfile.NamedTemporaryFile(suffix=".usda", prefix="tempTestStage", delete=True)
+        shutil.copy('./testMayaRefStageA.usda', tempPath.name)
```

Comment:
note: causing a Permissions error on windows... will fix soon.
```
IOError: [Errno 13] Permission denied: 'c:\\temp\\tempTestStagetuuicr.usda'
```

Reply:
OK found some info about the windows discrepancy [here](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) and pushed up a fix for the situation. I'm hopeful that the test will pass now.

Reply:
@kxl-adsk I'm not sure how to trigger the workflow so that I can see if the windows test is fixed. Is that something you have to do?